apps/data_handler/db/models/zklend_events.py

```diff
@@ -39,3 +39,80 @@ class LiquidationEventData(EventBaseModel):
     debt_face_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
     collateral_token: Mapped[str] = mapped_column(String, nullable=False)
     collateral_amount: Mapped[Decimal] = mapped_column(Numeric(38, 18), nullable=False)
+
+
+class RepaymentEventData(EventBaseModel):
+    """
+    Database model for Repayment event, inheriting from EventBaseModel.
+
+    This model stores details of a repayment event, including the addresses involved
+    and the amounts in raw and face formats.
+    """
+
+    __tablename__ = "repayment_event_data"
```

Comment:
we don't need to have here `_data`. please remove it
`repayment_event` will be okay